Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -557,3 +557,29 @@ extension SPAppDelegate {
         UserDefaults.standard.set(true, forKey: .hasMigratedSustainerPreferences)
     }
 }
+
+// MARK: - Content Recovery
+//
+extension SPAppDelegate {
+    @objc
+    func attemptContentRecoveryIfNeeded() {
+        let context = NSManagedObjectContext(concurrencyType: .privateQueueConcurrencyType)
+        context.persistentStoreCoordinator = coreDataManager.persistentStoreCoordinator
+
+        Task {
+            let restoredContent = await ContentRecoveryManager().prepareRecoveredNoteContentIfNeeded(in: context)
+            restoredContent.forEach({ insertNote(with: $0) })
```

Comment:
Just noticed in macOS, this is a Core Data threading violation. The Task is on a BG queue, and the Simperium API isn't threadsafe